configs/templates/ldap/ldap.yaml

```diff
@@ -3,14 +3,21 @@ apiVersion: v1
+  #passwords: cmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMQ==
+  #users: bGRhcC1hZG1pbjEsbGRhcC1hZG1pbjIsbGRhcC1hZG1pbjMsbGRhcC1hZG1pbjQsbGRhcC1hZG1pbjUsbGRhcC1hZG1pbjYsbGRhcC1hZG1pbjcsbGRhcC1hZG1pbjgsbGRhcC1hZG1pbjksbGRhcC1hZG1pbjEwLGxkYXAtYWRtaW4xMSxsZGFwLWFkbWluMTIsbGRhcC1hZG1pbjEzLGxkYXAtYWRtaW4xNCxsZGFwLWFkbWluMTUsbGRhcC1hZG1pbjE2LGxkYXAtYWRtaW4xNyxsZGFwLWFkbWluMTgsbGRhcC1hZG1pbjE5LGxkYXAtYWRtaW4yMCxsZGFwLXVzZXIxLGxkYXAtdXNlcjIsbGRhcC11c2VyMyxsZGFwLXVzZXI0LGxkYXAtdXNlcjUsbGRhcC11c2VyNixsZGFwLXVzZXI3LGxkYXAtdXNlcjgsbGRhcC11c2VyOSxsZGFwLXVzZXIxMCxsZGFwLXVzZXIxMSxsZGFwLXVzZXIxMixsZGFwLXVzZXIxMyxsZGFwLXVzZXIxNCxsZGFwLXVzZXIxNSxsZGFwLXVzZXIxNixsZGFwLXVzZXIxNyxsZGFwLXVzZXIxOCxsZGFwLXVzZXIxOSxsZGFwLXVzZXIyMCxsZGFwLW5vYWNjZXNzMSxsZGFwLW5vYWNjZXNzMixsZGFwLW5vYWNjZXNzMyxsZGFwLW5vYWNjZXNzNCxsZGFwLW5vYWNjZXNzNSxsZGFwLW5vYWNjZXNzNixsZGFwLW5vYWNjZXNzNyxsZGFwLW5vYWNjZXNzOCxsZGFwLW5vYWNjZXNzOSxsZGFwLW5vYWNjZXNzMTAsbGRhcC1ub2FjY2VzczExLGxkYXAtbm9hY2Nlc3MxMixsZGFwLW5vYWNjZXNzMTMsbGRhcC1ub2FjY2VzczE0LGxkYXAtbm9hY2Nlc3MxNSxsZGFwLW5vYWNjZXNzMTYsbGRhcC1ub2FjY2VzczE3LGxkYXAtbm9hY2Nlc3MxOCxsZGFwLW5vYWNjZXNzMTksbGRhcC1ub2FjY2VzczIwLGNvbGxpc2lvbi11c2VyLCBsZGFwLXNwZWNpYWw/LGxkYXAtc3BlY2lhbC4sbGRhcC1zcGVjaWFsXixsZGFwLXNwZWNpYWwkLGxkYXAtc3BlY2lhbCosbGRhcC1zcGVjaWFsKCxsZGFwLXNwZWNpYWwpLGxkYXAtc3BlY2lhbFssbGRhcC1zcGVjaWFsXSxsZGFwLXNwZWNpYWx7LGxkYXAtc3BlY2lhbH0sbGRhcC1zcGVjaWFsfCxsZGFwLXNwZWNpYWxA
+  # Password is rhodaPW@1
+  passwords: cmhvZGFQV0AxLHJob2RhUFdAMSxyaG9kYVBXQDEscmhvZGFQV0AxLHJob2RhUFdAMSxyaG9kYVBXQDEscmhvZGFQV0AxLHJob2RhUFdAMQ==
```

Comment:
How are we passing the credentials to login via Console?

Reply:
Not right now, currently we are just creating the LDAP IDP for login purpose. The plan is in place to integrate the created credentials into test-variables.yaml once we make use of ldap users in our automation.